src/main/perl/MD.pm

```diff
@@ -307,33 +310,18 @@ then
 EOC
     foreach my $dev (@{$self->{device_list}}) {
         $dev->create_ks;
-        # Well, fdisk sucks and Anaconda is plain crap. Let's
-        # guess how we can set the partition hex code
```

Comment:
well it is not using that dubious fdiks code anymore, and the parted 'set raid on' commands looks a bit more clearer, but I can ofcourse add an *explicit* comment here :smile: